### ISSUE TITLE
Create individual recruitment referral links for new members

### ISSUE THREAD

**Issue by TheBlackxRanger** [emperorshammer/issues-incoming-queue#156]
A suggestion from AD Impulse. Reinstating the ability to create referrals links to send to new members/prospects which would automatically assign the respective recruiter as a new recruit's recruiter on the join form. 

AD Impulse suggests this is a feature that used to be in place but is now disabled.

![Screenshot_20230305-131945](https://user-images.githubusercontent.com/74426625/222986659-4414afe2-3525-42e0-bc27-92fd4563e8f7.png)

**Comment by MrPliffles** [comment]
Encouraging members to spam their "referral link" in the hopes of earning a pile of MoIs isn't a good idea and there's a toll that the process of churning through unvetted applicants takes on the leaders of the club. If someone genuinely feels like a member led them to the TC, there's already an opportunity for them to name their recruiter on the form and I'd say that's enough.

**Comment by turtlexiv** [comment]
In general, I feel that sending people directly to the join form is poor practice and encourages low-quality recruiting efforts.  It's intended to be what it is (the form), not a landing page for prospective new members.  Other pages such as https://tc.emperorshammer.org/aboutus.php or https://tc.emperorshammer.org/recruitment.php are more intended for that purpose.

To that end, it would seem that incentivizing people to use a customized link directly to the join form isn't the best idea.  In concept we could catch a referral PIN on one of the other landing pages, and use it if the user happened to go to the join page from there (in fact, specifically NOT enable direct referral linking to the join page).  

Comments/thoughts welcome.

**Comment by TheBlackxRanger** [comment]
My only suggestion would be to make the "Would you.like to join our team?" wording larger or make a button like on the recruitment page so that it's very clear and enticing that it it is a "Join/sign up" button
Maybe rewording to say "would you like to be a member?" Or something to that effect 
 
 
  On Sat, Apr 1, 2023 at 5:20 PM, ***@***.***> wrote:   


In general, I feel that sending people directly to the join form is poor practice. It's intended to be what it is (the join form), not a landing page for prospective new members. Other pages such as https://tc.emperorshammer.org/aboutus.php or https://127.0.0.1/recruitment.php are more intended for that purpose.

To that end, it would seem that incentivizing people to use a customized link directly to the join form isn't the best idea. In concept we could catch a referral PIN on one of the other landing pages, and use it if the user happened to go to the join page from there (in fact, specifically NOT enable direct referral linking to the join page).

Comments/thoughts welcome.
